bin/hdfcoinc/pycbc_combine_coincident_events

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""
+This program combines together a set of STATMAP files from disjoint times.
+The resulting file would contain triggers from the full set of input files
+"""
+import numpy
```

Comment:
please remove double and/or unneeded imports.

Reply:
Done